src/styles/_catalog.scss

```
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: all, 300ms;
```

avoid using `all`
it slows down page load time
better specify certain ones
```suggestion
    transition: transform, 300ms;
```